src/SeleniumLibrary/keywords/screenshot.py

```diff
@@ -235,3 +237,20 @@ def _embed_to_log_as_file(self, path, width):
             f'<a href="{src}"><img src="{src}" width="{width}px"></a>',
             html=True,
         )
+    
+    @keyword
+    # def print_page_as_pdf(self, print_options: Optional[PrintOptions]=None,):
```

Comment:
Trying to figure out the proper type hints and type conversion signature for this method ..

Reply:
Initially accepting the all the print options within keyword arguments.